libs/sdk-core/src/breez_services.rs

```diff
@@ -834,6 +854,10 @@ impl BreezServicesBuilder {
                 .fiat_api
                 .clone()
                 .unwrap_or_else(|| breez_server.clone()),
+            moon_pay_api: Arc::new(Mutex::new(MoonPayApi::new(
+                self.config.moon_pay_config.clone(),
+                Box::new(breez_server.get_signer_client().await?),
```

Comment:
Other API traits are implemented in BreezServer (LspAPI, FiatAPI, SwapperAPI). For consistency perhaps it is better to add implementatino of the signer to BreezServer the same way and pass it to the MoonPayApi ?